_data-prepper/common-use-cases/sampling.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Sampling
+parent: Common use cases
+nav_order: 45
+---
+
+# Sampling
+
+Data Prepper provides the following sampling capabilities:
+
+- Time sampling
+- Percentage sampling
+- Tail sampling
+
+## Time sampling 
+
+You can use the `rate_limiter` action within the [`aggregate` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/aggregate/) to limit the number of events that can be processed per second. You can choose to either drop excess events or carry them forward to the next time period.
+
+In this example, only 100 events per second with a status code of `200` are sent to the sink from a given IP address. It drops all excess events from the configured time window.
```

Comment:
Confirm that my changes retain the intended meaning. Can we replace "It" in the second sentence with the noun?

Reply:
@natebower Revised to read: `The `when_exceeds` option is set to `drop`, which mean that  all excess events from the configured time window will be dropped.`

Reply:
@vagimeli How about "meaning" instead of "which means"? Otherwise, LGTM.